### ISSUE TITLE
Pop OS does not resume properly (i915 error) on any other desktop environment apart from Gnome

### ISSUE THREAD

**Issue by leminhman0312** [pop-os/pop#2640]
**Distribution:** 
NAME="Pop!_OS"
VERSION="22.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 22.04 LTS"
VERSION_ID="22.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=jammy
UBUNTU_CODENAME=jammy
LOGO=distributor-logo-pop-os

 
 
 
**Issue/Bug Description:**
Stock installation of Pop OS on Lemur Pro (lemp11).  Hibernation/resume works with default setup.  Hibernation seems to work with other desktop environment / window manager (XFCE, i3wm..).  Unfortunately, resume doesnt seem to work properly.  After authenticating with the luks encryption, the machine go straight to a black screen.  I figured this must be the screen where GDM is, because blindly typing the password and press Enter would seem to unlock the session.  After unlocking, I see the black screen again, so the actual desktop environment is also a  black screen.  Keyboard backlights work and volume control  also gives off sound to.  So, I am logged in, just not be able to see anything due to the black screen. 

**Steps to reproduce (if you know):**
1. Fresh installation of Pop OS 
2. Follow instructions here to enable hibernation: https://support.system76.com/articles/enable-hibernation/
3. Test to see if hibernation/resume works with stock installation. 
4. Download another desktop environment, say XFCE. 
5. Try to hibernate/resume again.  The black screen will show up. 

**Expected behavior:**
Regardless of desktop environments, hibernation/resume should work because it is a system wide settings, just like shutdown, reboot.  I should be able to resume from hibernation and see my files/window configuration are exactly where they are before hibernation.  I should be able to use my desktop environment properly (no black screen), just like before I attempted hibernation. 

**Other Notes:**
A quick look at the log file via journalctl shows the errors seem to be with i915:

```
Oct 08 13:03:49 pop-os systemd-journald[587]: Missed 118 kernel messages
Oct 08 13:03:49 pop-os kernel: i915 0000:00:02.0: [drm] *ERROR* Fault errors on pipe A: 0x00000080

```

There is a lot of these errors during the resume process.

**Comment by jacobgkau** [comment]
Just to be clear, it's only resume from hibernation that is not working on other desktops? Resume from suspend does work?

**Comment by leminhman0312** [comment]
Yes, suspend I can say always work.  I set mine to suspend on lid close and it always work, i.e. close lid, laptop goes to sleep (blinking yellow light on Lemur Pro), and upon open lid, my login manager shows up.  I then login and everything is normal. 

Resuming from hibernation, though, I can say that it 100% works with default Pop OS + default Pop OS's Gnome.  For other environment on Pop OS, it 100% never work (always resume into black screen).  

Hibernation itself seems working, because the laptop seems to be powered off after "systemctl hibernate"
